docs/implementation-guidelines.md

```diff
@@ -55,11 +55,10 @@ Modules must contain at least one automated validation test that runs Terraform
 
 ## Commit messages and versioning
 
-Use [Conventional Commit](https://www.conventionalcommits.org) messages when you commit code.
-
 Module changes adhere to [semantic versioning](https://semver.org/), with releases labeled as `{major}.{minor}.{patch}`. [Git tags](https://git-scm.com/book/en/v2/Git-Basics-Tagging) are used to identify the module versions. For more information about versioning, see [Release versioning](versioning.md).
 
-As a module author or contributor, you don't need to release a new version. Conventional commits determine whether a new version of a module is needed and which semantic versioning number to use. When a PR is merged into the main branch, a new release is created by the release automation if the validation pipeline passes. The commit messages are published in the release notes for the module.
+As a module author or contributor, you don't need to release a new version. The merging team determines whether a new version of a module is needed and which semantic versioning number to use based on the commits in the PR. When a PR is merged into the main branch, a new release is created by the release automation if the validation pipeline passes. The commit messages are published in the release notes for the module.
```

Comment:
I'm not sure we need any of this...

> The merging team determines whether a new version of a module is needed and which semantic versioning number to use based on the commits in the PR
^ This is not true - the merging team will determine if a version is needed based on the content of the PR, not the commits used. They may also check the checkboxes in the PR description, but that is not always accurately checked by the contributor. 

> When a PR is merged into the main branch, a new release is created by the release automation if the validation pipeline passes.
^ This is not true. There is no validation pipeline when a PR is merge. Just a release pipeline

> The commit messages are published in the release notes for the module.
^ Its actually the commit message used by the merging team when they squash and merge

So I think all of this content can be removed. And perhaps rename the title just `Versioning`

I have an action to create a merger team specific doc with more info on squash and merge process, but its not something that general contributors need to know about.

Reply:
OK. Removed that paragraph and the next about conventional commits that slipped by